tests/unit/data/test_dataset.py

```diff
@@ -55,13 +92,54 @@ def test_num_tasks(dataset, targets):
     assert dataset.t == targets.shape[1]
 
 
+@pytest.mark.skipif(
+    not all([x is None for x in ["X_d", "V_fs", "E_fs", "V_ds"]]), reason="Not all inputs are None"
+)
 def test_aux_nones(dataset: MoleculeDataset):
-    np.testing.assert_array_equal(dataset.X_f, None)
-    np.testing.assert_array_equal(dataset.X_f, None)
+    np.testing.assert_array_equal(dataset.X_d, None)
     np.testing.assert_array_equal(dataset.V_fs, None)
     np.testing.assert_array_equal(dataset.E_fs, None)
+    np.testing.assert_array_equal(dataset.V_ds, None)
     np.testing.assert_array_equal(dataset.gt_mask, None)
     np.testing.assert_array_equal(dataset.lt_mask, None)
-    assert dataset.d_vd == 0
+    assert dataset.d_xf == 0
     assert dataset.d_vf == 0
     assert dataset.d_ef == 0
+    assert dataset.d_vd == 0
+
+
+def test_normalize_targets(dataset):
+    dset_scaler = dataset.normalize_targets()
+    scaler = StandardScaler()
+    scaler.fit(dataset._Y)
+    Y = scaler.transform(dataset._Y)
+    
+    np.testing.assert_array_equal(dataset.Y, Y)
+    np.testing.assert_array_equal(dset_scaler.mean_, scaler.mean_)
+    np.testing.assert_array_equal(dset_scaler.scale_, scaler.scale_)
+
+
+def test_normalize_inputs(dataset):
+    dset_scaler = dataset.normalize_inputs("X_d")
+    scaler = StandardScaler()
+    scaler.fit(dataset._X_d)
```

Comment:
Does this need to be `scaler = scaler.fit(dataset._X_d`? I thought `.fit` returns the fitted scaler.

Reply:
The `fit` function does an in-place update and returns itself, so I don't think it is necessary

```
def fit(self, X, y=None, sample_weight=None):
        """Compute the mean and std to be used for later scaling.

        Parameters
        ----------
        X : {array-like, sparse matrix} of shape (n_samples, n_features)
            The data used to compute the mean and standard deviation
            used for later scaling along the features axis.

        y : None
            Ignored.

        sample_weight : array-like of shape (n_samples,), default=None
            Individual weights for each sample.

            .. versionadded:: 0.24
               parameter *sample_weight* support to StandardScaler.

        Returns
        -------
        self : object
            Fitted scaler.
        """
```